src/select-virtualized/shared-helpers/getters.js

```diff
@@ -11,7 +11,7 @@ export const getListHeight = ({
 export const getScrollIndex = ({ children, selected, valueGetter }) => {
   if (children && selected && valueGetter)
     return children.findIndex(
-      (child) => (valueGetter(child) || valueGetter(child.props.data)) === valueGetter(selected),
+      (child) => ((valueGetter(child) !== undefined) || valueGetter(child.props.data)) === valueGetter(selected),
```

Comment:
This is not what we need here. `((valueGetter(child) !== undefined) || valueGetter(child.props.data))` need to get the actual text so then when we compare it to `=== valueGetter(selected)` we compare is it is the same. what is the particular case that it fails for you?

It looks like there is something else is failing because of the empty value condition.

this is for a empty value or empty label?

Reply:
Oh, my bad, you are right. Let me update it. I tried too hard to cram the solution in a single line.

Reply:
What about `(child.props && valueGetter(child.props.data) || valueGetter(child))`? This does change the precedence but will return the correct value in both cases.
If you wish to keep the precedence of `valueGetter(child)` I don't see a single line solution but we can obviously do 
```let childValue = valueGetter(child);
    if(childValue === undefined){
      childValue = valueGetter(child.props.data);
    }
    return childValue === valueGetter(selected);
```
Which one would you prefer? (Or a different one)

Reply:
can you update the PR with the changes you mention, so i can see the whole picture?